packages/api/lib/errors.js

```diff
@@ -7,8 +7,14 @@ const {
   ValidationError,
 } = require('@cumulus/errors');
 
-const isBadRequestError = (err) =>
-  err.name === 'SchemaValidationError' || err instanceof ValidationError;
+const isPostgresValidationError = (error) => ['22', '23'].includes((error.code || '').substring(0, 2));
+
+const isBadRequestError = (error) =>
+  error.name === 'SchemaValidationError'
+  || error instanceof ValidationError
+  || isPostgresValidationError(error);
+  // Postgres error codes:
+  // https://www.postgresql.org/docs/10/errcodes-appendix.html
```

Comment:
This comment maybe makes more sense above `isPostgresValidationError` ?

Reply:
👍

Reply:
99cd413a0